client/my-sites/earn/home.tsx

```diff
@@ -198,23 +201,21 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 					},
 			  }
 			: {
-					text: translate( 'Collect payments' ),
+					text: ctaTitle,
 					action: () => {
 						trackCtaButton( 'recurring-payments' );
 						page( `/earn/payments/${ selectedSiteSlug }` );
 					},
 			  };
-		const hasConnectionTitle = translate( 'Manage payments' );
-		const noConnectionTitle = translate( 'Collect payments' );
-		const title = hasConnectedAccount ? hasConnectionTitle : noConnectionTitle;
+		const title = translate( 'Collect payments' );
 
 		const hasConnectionBody = translate(
 			"Manage your customers and subscribers, or your current subscription options and review the total revenue that you've made from payments."
 		);
 		const noConnectionBody = (
 			<>
 				{ translate(
-					'Accept one-time and recurring credit card payments for physical products, services, memberships, subscriptions, and donations.'
+					'Let visitors pay for digital goods and services or make quick, pre-set donations by enabling the Payment Button block.'
```

Comment:
I think the issue is saying that we ought to use the same wording whether or not Stripe is connected, though I think your approach here is better. I've queried [here](https://github.com/Automattic/wp-calypso/issues/62721#issuecomment-1112222715) so I can be sure we know what we're reviewing when ready

Reply:
Yup, the issue wants the same text for all versions, although I agree with you that it feels a bit awkward.
Let's wait for Seth's direction on this.

Reply:
Seth suggested this change:

> Let visitors pay for digital goods and services or make quick, pre-set donations by inserting the Payment Button block.